content/verification/veteran-verification/release-notes/2023-06-23

```diff
@@ -0,0 +1,14 @@
+We added an update to (v2) of the Veteran Service History & Eligibility to the developer portal, and the API has been renamed to the 'Veteran Service History and Eligibility API'.
+The following disability rating fields have been added:
+  -diagnostic_text
+  -disability_rating_id
+
```

Comment:
Since the previous release note gives all these disclaimers, I don't think we have to give them again for this next release note.

Reply:
Ok, I thought that was a required area to have for release notes.